centreon/cron/centreon-send-stats.php

```diff
@@ -142,15 +146,19 @@ function logger(string $message, Throwable $exception = null)
             'additional' => $additional
         );
 
-        $returnData = $http->call(CENTREON_STATS_URL, 'POST', $data, array(), true);
-        logger(
-            sprintf(
-                'Response from [%s] : %s,body : %s',
-                CENTREON_STATS_URL,
-                $returnData['statusCode'],
-                $returnData['body']
-            )
-        );
+        if ( isset($options["d"]) || isset($options["debug"]) ) {
```

Comment:
A few bytes less in the script 😄 
```suggestion
        if ( isset($options["d"], $options["debug"]) ) {
```